source/dom/dom.css

```diff
@@ -134,3 +134,9 @@ body .enyo-fullscreen {
 	height: 100% !important;
 	box-sizing: border-box !important;
 }
+
+/* Accessibility CSS */
+*[tabindex] {
+	/* remove outline in case dom has tabindex attribute */
+	outline: none;
```

Comment:
Tap highlight is only shown on the browser based on mobile webkit, so I think it should be added on garnet library.